Cargo.lock

```diff
@@ -5470,23 +5504,25 @@ checksum = "49874b5167b65d7193b8aba1567f5c7d93d001cafc34600cee003eda787e483f"
 [[package]]
 name = "vodozemac"
 version = "0.4.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "56d1b4c112e19e1f9361e34c7a40d30bd7f1de0fec43b83f2f8383d6eee9f561"
+source = "git+https://github.com/matrix-org/vodozemac/?rev=cedc9a08fdd8d0931ea778acf68c619a8298ee70#cedc9a08fdd8d0931ea778acf68c619a8298ee70"
 dependencies = [
  "aes",
  "arrayvec",
- "base64 0.13.1",
+ "base64 0.21.2",
  "cbc",
+ "curve25519-dalek",
  "ed25519-dalek",
+ "getrandom 0.2.10",
  "hkdf",
  "hmac",
  "matrix-pickle",
  "pkcs7",
  "prost",
- "rand 0.7.3",
+ "rand 0.8.5",
```

Comment:
So we still depend on old rand after this, ugh. Tracked that down and opened another upstream issue: https://github.com/LukeMathWalker/wiremock-rs/issues/123